src/python/TaskWorker/Actions/RetryJob.py

```diff
@@ -469,9 +469,9 @@ def check_corrupted_file(self, exitCode):
             proxy = os.getenv('X509_USER_PROXY')
             self.logger.info(f"X509_USER_PROXY = {proxy}")
             if corruptedFile:
-                reportLocation = 'gsiftp://eoscmsftp.cern.ch/eos/cms/store/temp/user/BadInputFiles/corrupted/new/'
+                reportLocation = ' davs://eoscms.cern.ch:443/eos/cms/store/temp/user/BadInputFiles/corrupted/new/'
```

Comment:
Any reason to have space before `dav://`?

Reply:
no. I did not noticed that in copy/paste I had introduced the space.  sorry. Let's remove

Reply:
resolved via https://github.com/dmwm/CRABServer/pull/8097/commits/bb934656599d4996467972c37d20841653790814